connector_ecommerce/views/sale_order.xml

```diff
@@ -46,10 +22,25 @@
                     groups="sales_team.group_sale_manager"
                 />
             </xpath>
+            <xpath expr="//notebook" position="inside">
+                <!-- change the invisible attribute to 0 when used in
+                    submodules -->
+                <page
+                    name="connector"
+                    string="Connectors"
+                    invisible="1"
+                    groups="connector.group_connector_manager"
+                >
+                    <group>
+                        <field name="canceled_in_backend" />
+                        <field name="cancellation_resolved" invisible="1" />
```

Comment:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0
_If the module brings from past versions fields in the views as invisible because they are used in expressions (invisible, column_invisible, readonly, required, context, domain) of other fields, you can now remove them for cleanliness, as they are auto-added. See more info at https://github.com/odoo/odoo/pull/137031._